src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -283,6 +283,19 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         // fuse softmax, MVN patterns, so that they will not be marked as precision sensitive in ConvertPrecision
         manager.register_pass<ov::pass::SoftmaxFusion>();
         manager.register_pass<ov::pass::MVNFusion>();
+
+        // fuse RMS patterns, so that they will not be marked as precision sensitive in ConvertPrecision
+        manager.register_pass<ov::pass::RMSFusion>(false);
```

Comment:
It seems that `force_tail_convert` parameter for this pass is not needed anymore as both plugins set `false` value